cmd/provision/README.md

```diff
@@ -1,77 +1,43 @@
-# Provisioning
-
-This command offers all required functionality to provision an host to join a Kubernetes cluster.
-
-The following operating systems are supported
-- Ubuntu 18.04
-- CentOS 7
-- ContainerLinux / CoreOS (Not tested with RedHat CoreOS)
-
-## Requirements
-- The cluster needs to use the bootstrap token authentication
-
-## CLI
-
-```bash
-./provision \
-    --kubelet-version="v1.13.1" \
-    --cloud-provider="openstack" \
-    --cloud-config="/etc/kubernetes/cloud-config" \
-    --token="AAAAAAAAAAAAAAAA" \
-    --ca-cert="/etc/kubernetes/ca.crt"
-```
-
-## Process
-
-Nodes will boot with a cloud-init (Or Ignition) which writes required files & a shell script (called `setup.sh` here).
-
-### cloud-init (Or ignition)
-Parts which will be covered by cloud-init (or Ignition)
-
-- Install SSH keys
-- Configure hostname
-- `ca.crt`
-    The CA certificate which got used to issue the certificates of the API server serving certificates
-- `cloud-config`
-    A optional cloud-config used by the kubelet to interact with the cloud provider.
-- `setup.sh`  
-    Is responsible for downloading the `provision` binary and to execute it.
-    The download of the binary might also be done using built-in `cloud-init` (or Ignition) features
-
-### Provision
-
-The `provision` binary will identify the operating system and execute a set of provisioning steps.
-
-The provisioning process gets separated into 2 phases:
-- Base provisioning
-  Install and configure all required dependencies
-- Join
-  Write & start the kubelet systemd unit
-
-#### Base provisioning
-The following steps belong into the base provisioning:
-- Install required packages (apt & yum action)
+## Provision
+The `provision` will identify the operating system and execute a set of
+provisioning steps. It helps to provision a host exacly the same way as
+machine-controller would do.
+
+## Use-cases
+- Prepare VM images for machine-controller
+- (In future) Join host to existing Kubernetes cluster using bootstrap token
```

Comment:
Kinda yes. It was specified in original doc as a meat to save user-data space to work around its limit. For now we are good with gzip, but this can end after some additions.